server/serializer/channel_subscription.go

```diff
@@ -72,6 +74,20 @@ type Subscriptions struct {
 	BySpaceID     map[string]string
 }
 
+type OldSubscriptions struct {
+	ByChannelID   map[string]StringSubscription
+	ByURLPagID    map[string]StringArrayMap
```

Comment:
can't change this because this is the same in the previous confluence plugin and I can't change this as this would change the name of the key stored in KV store